packages/core/src/modules/proofs/formats/indy/IndyProofFormat.ts

```diff
@@ -59,19 +63,10 @@ export interface IndyProofFormat extends ProofFormat {
     requestCredentials: IndyRequestedCredentialsFormat
     retrieveCredentials: IndyRetrievedCredentialsFormat
   }
-  // Format data is based on RFC 0592
-  // https://github.com/hyperledger/aries-rfcs/tree/main/features/0592-indy-attachments
-  // formatData: {
-  //   proposal: {
-  //     schema_issuer_did?: string
-  //     schema_name?: string
-  //     schema_version?: string
-  //     schema_id?: string
-  //     issuer_did?: string
-  //     cred_def_id?: string
-  //   }
-  //   offer: CredOffer
-  //   request: CredReq
-  //   credential: Cred
-  // }
+
+  formatData: {
+    proposal: IndyProofProposal
+    request: IndyProofRequest
```

Comment:
This should both be `IndyProofRequest` imported form the `@types/indy-sdk` package (so `import type { IndyProofRequest } from 'indy-sdk'`, not the class instance (as we're returning the json)